lib/apiservers/engine/backends/container.go

```diff
@@ -1377,14 +1377,22 @@ func validateCreateConfig(config *types.ContainerCreateConfig) error {
 	config.HostConfig.Memory = memoryMB
 	log.Infof("Container memory: %d MB", config.HostConfig.Memory)
 
-	if config.NetworkingConfig == nil {
-		config.NetworkingConfig = &dnetwork.NetworkingConfig{}
-	}
-
 	if config.HostConfig == nil || config.Config == nil {
 		return BadRequestError("invalid config")
 	}
 
+	if config.NetworkingConfig == nil {
```

Comment:
There is one file "backends/container_test.go" which I think does unit test for this file backends/container.go. But I don't think there is a test which creates a container without NetworkingConfig. Should I add test code?